PetGame/Controllers/PetController.cs

```diff
@@ -50,6 +50,34 @@ public IActionResult Get(ulong id)
             return Json(pet);
         }
 
+        /// /api/Pet/[PetId]/status (No Request Body)
+        /// <summary>
+        /// Returns the status of a single pet by its ID
+        /// </summary>
+        /// <param name="Request">
+        /// JSON request body containing the Pet's ID
+        /// </param>
+        /// <returns>
+        /// JSON of PetStatus object
+        /// </returns>
+        [HttpGet("{id}/status"), AllowAnonymous]
+        public IActionResult Get(StatusRequest Request)
```

Comment:
You are on the right track, but we shouldn't box the ID into it's own object. The names of parameters have to match that in the route definition, which is (probably) why you were getting the conflicting issue as described in StatusRequest.
```suggestion
        public IActionResult Get(ulong id)
```